app/components/Exchange/Exchange.jsx

```diff
@@ -344,7 +344,7 @@ class Exchange extends React.Component {
             hideScrollbars: ws.get("hideScrollbars", false),
             singleColumnOrderForm: ws.get("singleColumnOrderForm", true),
             flipOrderBook: ws.get("flipOrderBook", false),
-            flipBuySell: ws.get("flipBuySell", false),
+            flipBuySell: ws.get("flipBuySell", true),
```

Comment:
out of scope

Reply:
I know, do you have a concern for making it true?